docs/changelog.md

```diff
@@ -8,6 +8,7 @@ Change Log
 * Fix: Preserve nullability in `KSType.toClassName()`. (#1956)
 * New: Add `KSTypeAlias.toClassName()`. (#1956)
 * New: Add `KSType.toClassNameOrNull()`. (#1956)
+* Fix: Rename `interop-kotlinx-metadata.md` to `interop-kotlin-metadata.md` to fix link in docs (#1961)
```

Comment:
```suggestion
```
Since this is our infrastructure and nothing to care about for consumers, dropping the change log entry.

Reply:
Also the site deploys instantly and not with our releases, so your fix is already live!